### DISCUSSION TITLE
Is this remake going to be exactly the same layout, treasure etc?

### DISCUSSION THREAD

**Discussion by ionetrate** [davemoore22/sorcery#1] [General]
I was wondering if this was a redux or a duplicate?   Either way, this is quite a feat.

**Comment by ionetrate** [comment]
????

**Reply by Clauvin** [reply]
As far as I'm aware, same maps, but with options to tweak rules and game mechanics.

Also, check the DungeonCrawlers.org community at Discord, you can reach the developer of the project at #wizardry.

**Reply by ionetrate** [reply]
thank you for the reply.  I will check out the discord link!

**Reply by ionetrate** [reply]
he has made the the discord server private.

**Reply by Clauvin** [reply]
Ok, hang on a sec

**Reply by Clauvin** [reply]
I have a friend in the server, will ask him to see if he can get an invite

**Reply by Clauvin** [reply]
Just to be sure - https://discord.gg/XerEseQ - this link is not working?

**Reply by ionetrate** [reply]
ty

**Reply by davemoore22** [reply]
Oh Btw, apologies for the late answer to your questions, I didn't receive any notifications that you had posted here, and only came across your questions yesterday.

**Comment by davemoore22** [comment]
Yes, completely the same!

**Comment by ionetrate** [comment]
I appreciate the hard work you have put into it. There was a scenario editor out at the time wizardry 2 came out. Did you know that? 
Sent from Yahoo Mail on Android 
 
  On Thu., 17 Aug. 2023 at 7:30 p.m., Dave ***@***.***> wrote:   


Yes, completely the same!

**Reply by davemoore22** [reply]
I didn't.

The plan is to implement Wiz 1, then Wiz 2, and Wiz 3. As seperate modules or similar (not exactly sure yet how I will implement it).

Then before I do Wiz 5., I'll create a random procgen mode.

**Comment by ionetrate** [comment]
That is ok, you are busy and busy is good.  I was hoping you were going to create a brand new set of Wizardry games.  I have been waiting for something for years, but nothing.
    On Friday, August 18, 2023 at 09:19:16 a.m. EDT, Dave Moore ***@***.***> wrote:  
 
 


Oh Btw, apologies for the late answer to your questions, I didn't receive any notifications that you had posted here, and only came across your questions yesterday.

**Reply by davemoore22** [reply]
I believe Wizardry: Legacy of the Five Ordeals is a whole load of new scenarios and it's currently on Steam.

I have not played it because I dare not, or else I'll never get Sorcery finished :) 

I am going to go for a more rogue like or Diablo 3 approach for the random infinite content.

**Reply by Clauvin** [reply]
@ionetrate , other options currently available are:

* the Wandroid series for Android (all in english, all free with iap to remove ads, if I remember correctly);
* Buriedbornes (it's not a crawler as Wizardry, but oh boy that game does nails an opressive atmosphere and is easy to play on mobile - it's also available at Steam);
* Strangers of Sword City (it's hard, and it does that trick where enemies can be way stronger than what your party can deal with);
* Cursed Dungeon Unexplored (looks good... but first you have to make it work on your PC :/);

**Comment by ionetrate** [comment]
I have played all of the wizardry game up to 8.  I bought knight of diamonds when I was just 13, so I have had a long history with the game.  The only game that has scratched the itch has been Grimrock, but even Grimrock had too many puzzles and and only some of the magic that made the original wizardry great.  I was hoping that someone would start releasing new scenarios, that would really be amazing.
    On Friday, August 18, 2023 at 06:40:12 p.m. EDT, Cláuvin ***@***.***> wrote:  
 
 


@ionetrate , other options currently available are:
   
   - the Wandroid series for Android (all in english, all free with iap to remove ads, if I remember correctly);
   - Buriedbornes (it's not a crawler as Wizardry, but oh boy that game does nails an opressive atmosphere and is easy to play on mobile - it's also available at Steam);
   - Strangers of Sword City (it's hard, and it does that trick where enemies can be way stronger than what your party can deal with);
   - Cursed Dungeon Unexplored (looks good... but first you have to make it work on your PC :/);

**Comment by ionetrate** [comment]
There is a spelling error in the video sample :   
you used "distrubed instead of disturbed"

**Reply by davemoore22** [reply]
Oh my sense of spelling is most disturbed as is, thank you, I'll fix that.
